components/Animations/src/Enums/FrameworkLayer.cs

```diff
@@ -10,12 +10,20 @@ namespace CommunityToolkit.WinUI.Animations;
 public enum FrameworkLayer
 {
     /// <summary>
+#if WINUI2
     /// Indicates the <see cref="Windows.UI.Composition"/> APIs.
+#elif WINUI3
+    /// Indicates the <see cref="Microsoft.UI.Composition"/> APIs.
+#endif
```

Comment:
We could put this at the top with the using:
```csharp
#if WINUI2
using Composition = Windows.UI.Composition;
#elif WINUI3
using Composition = Microsoft.UI.Composition;
#endif
```
And then just be able to do `Composition` in the comment vs. duplicating all the text, same with below and anywhere else. Makes it easier to maintain the comment in the future.